crates/types/src/v1alpha1/wasmcloud_host_config.rs

```diff
@@ -22,7 +22,7 @@ pub struct WasmCloudHostConfigSpec {
     pub host_replicas: u32,
     /// A list of cluster issuers to use when provisioning hosts. See
     /// https://wasmcloud.com/docs/deployment/security/zero-trust-invocations for more information.
-    pub issuers: Vec<String>,
+    pub issuers: Option<Vec<String>>,
```

Comment:
We should probably update the comment above to note that this option is deprecated as of wasmCloud 1.0